.claude/commands/ship.md

```
@@ -0,0 +1,87 @@
+
+```bash
+gh issue view <n> --json number,title,body,url
+```
+
+## 2. Commit
+
+Message format is **`area: what changed`**.
+
+`area` is the part of the system touched — free-form, lowercase, whatever describes it best. Real examples from this repo's history:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)